src/js/Text/index.ts

```diff
@@ -0,0 +1,57 @@
+import Control, {ControlProps} from '../Control';
+import { element, any } from 'prop-types';
+
+type TextProps = ControlProps & {
+  value: string;
+}
+
+class Text extends Control {
+  protected _props: TextProps = {
+    ...this._props,
+    ...{
+      value: ''
+    }
+  }
+  private inputEl: any;
```

Comment:
Please, change type from 'any' to 'HTMLInputElement'. Generally please, avoid using 'any' as much as possible since we are writing in TypeScript now. Thanks!